core-plugins/src/main/java/co/cask/hydrator/plugin/common/StructuredToTextTransformer.java

```diff
@@ -0,0 +1,48 @@
+
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.hydrator.plugin.common;
+
+import co.cask.cdap.api.data.format.StructuredRecord;
+import co.cask.hydrator.common.RecordConverter;
+
+import java.io.IOException;
+
+/**
+ * Creates Single String from StructuredRecord
+ */
+public class StructuredToTextTransformer extends RecordConverter<StructuredRecord, String> {
```

Comment:
don't think there is any need to extend RecordConverter, since its not using any of the methods in RecordConverter